docs/modeling/new.rst

```diff
@@ -70,7 +70,7 @@ supported by the basic `~astropy.modeling.custom_model` factory function.
 
 The details are explained below with a 1-D Gaussian model as an example.  There
 are two base classes for models. If the model is fittable, it should inherit
-from `~astropy.modeling.FittableModel`; if not it should subclass
+from `~astropy.modeling.Fittable1DModel`; if not it should subclass
```

Comment:
I think this statement is meant to be general enough to include 2D, etc. Since it does not affect the actual example, there is no need to change this?